diff -urN kmix-4.13.2/gui/viewdockareapopup.cpp kmix-4.13.2-patched/gui/viewdockareapopup.cpp
--- kmix-4.13.2/gui/viewdockareapopup.cpp	2014-05-18 06:24:36.000000000 +1100
+++ kmix-4.13.2-patched/gui/viewdockareapopup.cpp	2014-06-14 22:12:50.235392723 +1100
@@ -211,7 +211,8 @@
 	delete layout(); // BKO 299754
 	_layoutMDW = new QGridLayout(this);
 	_layoutMDW->setSpacing(KDialog::spacingHint());
-	_layoutMDW->setMargin(0);
+	// Add some space over device icons, otherwise they may hit window border
+	_layoutMDW->setContentsMargins(0,5,0,0);
 	//_layoutMDW->setSizeConstraint(QLayout::SetMinimumSize);
 	_layoutMDW->setSizeConstraint(QLayout::SetMaximumSize);
 	_layoutMDW->setObjectName(QLatin1String("KmixPopupLayout"));
