diff -urN kmix-4.10.3/gui/viewdockareapopup.cpp kmix-4.10.3-patched/gui/viewdockareapopup.cpp
--- kmix-4.10.3/gui/viewdockareapopup.cpp	2013-05-03 15:41:29.153438148 +1100
+++ kmix-4.10.3-patched/gui/viewdockareapopup.cpp	2013-05-23 02:25:59.254528560 +1100
@@ -208,7 +208,8 @@
 //	delete _layoutMDW;
 	_layoutMDW = new QGridLayout(this);
 	_layoutMDW->setSpacing(KDialog::spacingHint());
-	_layoutMDW->setMargin(0);
+	// Add some space over device icons, otherwise they may hit window border
+	_layoutMDW->setContentsMargins(0,5,0,0);
 	//_layoutMDW->setSizeConstraint(QLayout::SetMinimumSize);
 	_layoutMDW->setSizeConstraint(QLayout::SetMaximumSize);
 	_layoutMDW->setObjectName(QLatin1String("KmixPopupLayout"));
